statick_tool/statick.py

```diff
@@ -530,7 +568,12 @@ def run_workspace(
                 mp_args.append((parsed_args, count, package, num_packages))
 
             with multiprocessing.Pool(parsed_args.max_procs) as pool:
-                total_issues = pool.starmap(self.scan_package, mp_args)
+                total_issues, all_timings = zip(  # type: ignore
```

Comment:
I had to ignore the type hint here, couldn't figure it out. Any insight or fixes would be nice.